pages/06_embedding_non_view_content.md

```diff
@@ -2,22 +2,64 @@
 title: Embedding Non-View Content
 ---
 
-Almost all embedded deployments of Tableau embed pre-built dashboards. But many embed other components of Tableau Server such as:
+Almost all embedded deployments of Tableau embed pre-built dashboards. But some embed other components of Tableau such as:
 
-* Web Authoring screens
+* web authoring views
 * The Tableau Server UI (embedded for integration and single sign-on reasons)
 
-There are no APIs to achieve this type of embedding, but you can use iframes and any of the [single sign-on capabilities]({{ site.baseurl }}/pages/02_auth_and_sso) to achieve this.
+The easiest and best way to include web authoring in your embedding scenario is to use the [Tableau Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html). The Tableau Embedding API v3 provides a web component, `<tableau-viz-authoring>`, which in just a few lines of HTML code, allows users to author the view in your web application. You can also use the Embedding API v3 to call JavaScript methods that let you dynamically configure and switch between the authoring or viewing experience.
 
 ## Embedding Web Authoring
 
-Web Authoring is the Tableau Desktop-like functionality in the browser. The fact that Web Authoring exists allows you to pass on the self-service capabilities of Tableau to your end users. You may have pre-built dashboards for them to use, but they may have minor customization request or may want to build visualizations and dashboards from scratch.
+Web authoring allows someone to edit a visualization just as they would in Tableau Desktop.
+Embedding a view with web authoring gives your users the ability to modify and update a view, all while staying in their workflow. Embedded web authoring means that you can create comprehensive embedded applications for your customers that now include options for self-servicing.
 
-For many situations, it is acceptable to simply allow the default behavior of allowing the user to click 'Edit' on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, if you want to keep the user inside your application, you may prefer to embed the web authoring capabilities. To do that:
+When it comes to editing a visualization, it is often acceptable to simply use the default settings and allow the user to click **Edit** on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, if you want to keep the user inside your application, you might prefer to embed the web authoring capabilities. To do that:
```

Comment:
Now they can embed Tableau authoring experience directly from the application, no need to jump to another window and tab. They can use the button in the toolbar or create their own button that match their own brand

Reply:
Good suggestion. Updated paragraph: 

When it comes to editing a visualization, it is often acceptable to simply use the default settings and allow the user to click **Edit** on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, now you can embed the Tableau authoring experience directly in the application. There's no need to have users jump to another window or tab. You can use the **Edit** button in the toolbar, or create your own edit button that matches the look and feel of your application. To do that: